src/sage/rings/lazy_series.py

```diff
@@ -6673,6 +6675,60 @@ def coefficient(n):
 
     compositional_inverse = revert
 
+    def legendre_transform(self):
+        r"""
+        Return the Legendre transform of ``self``.
+
+        Given a symmetric function `f` of valuation 2, the Legendre
+        transform of `f` is the unique symmetric function `g` of
+        valuation 2 over the same base ring, such that
+
+        .. MATH::
+
+            g \circ \partial_{p_1} f + f = p_1 \partial_{p_1} f.
+
+        This implies that the derivatives of `f` and `g` w.r.t. `p_1`
```

Comment:
```suggestion
        This implies that the derivatives of `f` and `g` with respect to `p_1`
```
Good to spell it out IMO.